src/RouterConfig.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Navbar, Nav, NavItem, NavDropdown, MenuItem, FormGroup, Form, FormControl, Grid, Row, Col, Button } from 'react-bootstrap'
+import { Link, Switch, Redirect, Route, BrowserRouter } from 'react-router-dom'
+import { LinkContainer } from 'react-router-bootstrap'
+import './App.css';
+import 'bootstrap/dist/css/bootstrap.min.css'  // temporary, we can create our css later
+import AppointmentHome from './Appointment/AppointmentHome'
+import ServiceHome from './Service/ServiceHome'
+import RequestHome from './Request/index'
+import CreateRequest from './Request/CreateRequest';
```

Comment:
There is no CreateRequest file or component

Reply:
Right, I did not request new PR for this :) 

Will be soon updated.